assets/quick-order-list.css

```diff
@@ -450,6 +450,10 @@ quick-order-list-remove-button:hover .icon-remove {
   }
 }
 
+.quick-order-list__button-text {
+  text-align: center;
```

Comment:
For QoL it doesn't matter, the button always wraps to a new line before the text gets squished enough for the bug to appear. But I figure it's better to keep the CSS consistent for both in case we change the wrapping point someday. What do you think?

Reply:
That is fair :) I am aligned to include it